tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/BaseTest__should_not_replace_tickets_block_on_events_not_in_series__0.snapshot.html

```diff
@@ -30,6 +30,7 @@ <h3 class="tribe-common-h7 tribe-tickets-notice__title">Checkout Unavailable!</h
 
 		<div
 	id="tribe-block-tickets-item-{{ticket_2}}"
+	data-seat-labels=""
```

Comment:
Per my comment above, this snapshot should not change when the Seating feature is not active.